client/interop/musicxml/importer/importer.go

```diff
@@ -353,25 +353,37 @@ func ImportMusicXML(r io.Reader) ([]model.ScoreUpdate, error) {
 
 	_, err := doc.ReadFrom(r)
 	if err != nil {
-		return nil, err
+		return nil, help.UserFacingErrorf("Failed to read from input.")
 	}
 
 	scorePartwise := doc.SelectElement("score-partwise")
 	scoreTimewise := doc.SelectElement("score-timewise")
 
 	if scorePartwise == nil && scoreTimewise != nil {
+		// The vast majority of applications export directly to score-partwise.
+		// Applying the XSLT here in Go would require many additional deps, and
+		// the user should be able to do so themselves somewhat easily.
 		return nil, help.UserFacingErrorf(
-			`Issue importing MusicXML file: please convert to %s instead of %s using XSLT before importing`,
-			color.Aurora.BrightYellow("score-partwise"),
+			"Issue importing MusicXML: please convert input from %s to %s format (MusicXML provides an XSLT stylesheet to perform this conversion)",
 			color.Aurora.BrightYellow("score-timewise"),
+			color.Aurora.BrightYellow("score-partwise"),
 		)
 	} else if scorePartwise == nil {
 		return nil, help.UserFacingErrorf(
-			`Issue importing MusicXML file: could not last %s root tag`,
+			"Issue importing MusicXML: could not find %s root tag",
 			color.Aurora.BrightYellow("score-partwise"),
 		)
 	}
 
+	version := scorePartwise.SelectAttrValue("version", "")
+	if version != "3.1" {
+		log.Warn().Msg(fmt.Sprintf(
+			`MusicXML import supports version %s. Behaviour may be undefined with provided version %s.`,
+			color.Aurora.BrightYellow("3.1"),
+			color.Aurora.BrightYellow(version),
+		))
+	}
```

Comment:
I agree that it's good to log a warning, since we only support 3.1 at the moment :+1: